scripts/sct_compute_hausdorff_distance.py

```diff
@@ -431,56 +431,62 @@ def non_zero_coord(data):
 
 def get_parser():
     # Initialize the parser
-    parser = Parser(__file__)
-    parser.usage.set_description('Compute the Hausdorff\'s distance between two binary images which can be thinned (ie skeletonized)'
-                                 'If only one image is inputted, it will be only thinned')
-    parser.add_option(name="-i",
-                      type_value="file",
-                      description="First Image on which you want to find the skeleton",
-                      mandatory=True,
-                      example='t2star_manual_gmseg.nii.gz')
-    parser.add_option(name="-d",
-                      type_value="file",
-                      description="Second Image on which you want to find the skeleton",
-                      mandatory=False,
-                      default_value=None,
-                      example='t2star_manual_gmseg.nii.gz')
-    parser.add_option(name="-r",
-                      type_value=None,
-                      description="Second Image on which you want to find the skeleton",
-                      mandatory=False,
-                      deprecated_by='-d')
```

Comment:
@YangHee-Min : I was wondering: Is there a way to have this option (ie `deprecated_by`) in the new version?

Reply:
i think we can remove all the "deprecated_by" in 4.0.0

Reply:
@charleygros I'm still looking for documentation but did not find any up till now. Will keep you posted